run_zephir_full_monthly.sh

```diff
@@ -26,42 +45,37 @@ echo "`date`: zephir full extract started"
 
 echo "`date`: retrieve zephir files: groove_export_${zephir_date}.tsv.gz"
 
-$ROOTDIR/ftpslib/ftps_zephir_get exports/$ZEPHIR_GROOVE_EXPORT $ZEPHIR_GROOVE_EXPORT
+run_external_command $ROOTDIR/ftpslib/ftps_zephir_get exports/$ZEPHIR_GROOVE_EXPORT $ZEPHIR_GROOVE_EXPORT
 if [ ! -e $ZEPHIR_GROOVE_EXPORT ]; then
-  echo "***" 
-  echo "file $ZEPHIR_GROOVE_EXPORT not found, exiting" 
-  echo "***" 
-  exit
+  report_error_and_exit "file $ZEPHIR_GROOVE_EXPORT not found, exiting" 
 fi
 
-mv $ZEPHIR_GROOVE_EXPORT $INGEST_BIBRECORDS/groove_full.tsv.gz
+run_external_command mv $ZEPHIR_GROOVE_EXPORT $INGEST_BIBRECORDS/groove_full.tsv.gz
 
 echo "*** retrieve full zephir vufind extract" 
 
-$ROOTDIR/ftpslib/ftps_zephir_get exports/$ZEPHIR_VUFIND_EXPORT $ZEPHIR_DATA/$ZEPHIR_VUFIND_EXPORT
+run_external_command $ROOTDIR/ftpslib/ftps_zephir_get exports/$ZEPHIR_VUFIND_EXPORT $ZEPHIR_DATA/$ZEPHIR_VUFIND_EXPORT
 if [ ! -e $ZEPHIR_DATA/$ZEPHIR_VUFIND_EXPORT ]; then
-  echo "***" 
-  echo "file $ZEPHIR_DATA/$ZEPHIR_VUFIND_EXPORT not found, exitting" 
-  echo "***" 
-  exit
+  report_error_and_exit "file $ZEPHIR_DATA/$ZEPHIR_VUFIND_EXPORT not found, exiting" 
 fi
 
 echo "*** dump the rights db to a dbm file" 
-$ROOTDIR/bld_rights_db.pl -x $RIGHTS_DBM
+run_external_command $ROOTDIR/bld_rights_db.pl -x $RIGHTS_DBM
 
 # split the json file 
 rm -f zephir_full_monthly_??
-echo input file is $ZEPHIR_DATA/$ZEPHIR_VUFIND_EXPORT 
+echo "input file is $ZEPHIR_DATA/$ZEPHIR_VUFIND_EXPORT"
 ls -l $ZEPHIR_DATA/$ZEPHIR_VUFIND_EXPORT
 
-unpigz -c $ZEPHIR_DATA/$ZEPHIR_VUFIND_EXPORT | split -l $SPLITCOUNT - zephir_full_monthly_
+run_external_command "unpigz -c $ZEPHIR_DATA/$ZEPHIR_VUFIND_EXPORT | split -l $SPLITCOUNT - zephir_full_monthly_"
```

Comment:
At this point I'm happy with any option that seems to work as expected. Overall this gets into a lot of hairy questions about shell parsing & quoting that confuse me a lot any time I try to figure them out, but I think so long as we have run it and verified that it works as expected.

That all said, if this doesn't work I think we have a couple options:

1) revert to quoting everything
2) another potential option to try would be `run_external_command ( the | whole | pipeline > outfile )`

for example

```bash
run_external_command (unpigz -c $ZEPHIR_DATA/$ZEPHIR_VUFIND_EXPORT | split -l $SPLITCOUNT - zephir_full_monthly_)
```